_ml-commons-plugin/tutorials/semantic-search-byte-vectors.md

```diff
@@ -0,0 +1,314 @@
+---
+layout: default
+title: Semantic search using byte vectors
+parent: Tutorials
+nav_order: 10
+---
+
+# Semantic search using byte quantized vectors
```

Comment:
Should the titles match, or is it shortened for navigation menu?

Reply:
Shortened for left nav :)